### DISCUSSION TITLE
RaspberryPi 3 with current versión?

### DISCUSSION THREAD

**Discussion by diegocesaretti** [geezacoleman/OpenWeedLocator#60] [Q&A]
Due to chip shortages is has become expensive un My country to get Rpi4, would the current revisión of OWL work fine on a Rpi3? Any limitations?

**Comment by geezacoleman** [comment]
The first version of the OWL was done using a RPi 3B+ and it worked ok, but it will be a bit slower processing frames and you need to be careful with exceeding the lower RAM. So if you can only access earlier versions of the Pi you can likely get it running. Looking at a comparison of the [different models](https://socialcompare.com/en/comparison/raspberrypi-models-comparison), the 3B+ has 1GB RAM with 1.4GHz CPU. The 4 B models have a 1.5GHz CPU with between 1 - 8GB RAM. Biggest change you might need to make is reducing image resolution.

It looks like Raspberry Pi supply issues may resolve later next year according to [Eben Upton recently](https://www.tomshardware.com/news/raspberry-pi-ceo-talks-shortages-next-gen-pi), so hopefully that will make building OWLs a little easier. Alternatively the [Rock Pi are more available](https://rockpi.org/) and look like they could work, but we haven't got around to testing them. So if you wanted to take on a project that could be a neat addition to the community!

**Reply by geezacoleman** [reply]
@williamtsalter just sent [me this article](https://www.raspberrypi.com/news/supply-chain-update-its-good-news/) through from Eben Upton with an update on the supply issues. Looks like they are easing a touch before Christmas and hopefully more so next year.
